cfp.md

```diff
@@ -1,9 +1,64 @@
+**What:** BSidesSF 2020 Call For Participation for Talks and Workshops
 
-As always -- Questions? E-mail cfp [at] bsidessf.org
+BSidesSF is a non-profit organization designed to advance the body of Information Security knowledge, by providing an annual, open forum for discussion and debate for security practitioners. We produce a conference that is a source of education, collaboration, and continued conversation for information technologists. The technical and academic presentations at BSidesSF are given in the spirit of peer review and advanced knowledge dissemination. This allows the field of Information Security to grow in breadth and depth, and continue in its pursuit of highly advanced, scientifically based knowledge. 
+
+We are a practitioner focused event that attracts a wide range of attendees ranging from engineers & security leaders to executive decision makers. This is a great opportunity for you to showcase your work and share your ideas with industry peers. Bonus points if you can tie your presentation to this year’s conference theme: San Francisco! 
+
+The BSides community has continuously raised the bar and put the info back in infosec. We thank each and every member/participant/organizer of this community for their hard work, sweat, and relentless pursuit of making high quality content accessible. We celebrate diverse approaches and points of view and believe that our commitment to diversity fosters innovation. BSidesSF is continuing to build upon a culture that drives our community to include a wide range of thoughts and ideas.
+
+**When:**  BSidesSF will take place in San Francisco from February 22-24, 2020.  The Call for Participation for talks and workshops will close on November 6, 2019 at 11:59 PM PST.
+
+**How** 
+This year, we are expanding the type of submissions we are supporting at BSidesSF. New session types include deep dive talks, lightning talks and an off the record track. 
+
+**Workshops: February 22, 2020** 
+Please note that workshops will be held on Saturday, February 22, 2020 at a different location.
+
+- **Condensed workshops** - 2 hr 45 mins
+- **Full length workshops** - 5 hr 30 mins 
+
+**Talks: February 23-24, 2020**
+Please note that all talks are limited to two speakers only. Panels are limited to 5 participants (including the moderator).
+ 
+- **General talks** - 25 min regular talk format for various topics in security and privacy.
+- **Deep dive talks** - 50 min talk format to cover more technical deep dives.
+- **Panels** -  50 min panel discussion to bring together a group of experts on a broadly accessible topic.
+- **Lightning talks** - 10 min lightning talk about a new idea or project that you are exploring.
+- **Off the record** - 25 min or 50 min talks for topics that you would like to discuss “Off the record”. We will be enforcing no recording/photography for this track.   
```

Comment:
Change the quotes to normal quotes.

Reply:
Fantastic eye! I blame Google Docs. Updated to normal quotes.